test/library/standard/Path/victor-ludorum/joinPath/testcases.good

```diff
@@ -0,0 +1,9 @@
+/baz
+/foo/./baz
+/foo/./baz
+/foo/baz
+/baz
+/foo/./baz
+/foo/baz
+/foo
```

Comment:
I think this line should instead be "/foo/".  Did you run this test through start_test to check?

Reply:
Yeah Mam I have mistakenly added test for joinPath("/foo/") . I will correct it by adding test for 
joinPath("/foo")  not for joinPath("/foo/") .